File: android/app/src/main/java/com/myjobpitch/api/data/Business.java
package com.myjobpitch.api.data;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.myjobpitch.api.MJPObjectWithDates;

import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Business extends MJPObjectWithDates {
    private String name;
    private List<Integer> users;
    private List<Integer> locations;
    private List<Image> images;
    private Integer  tokens;
    private Boolean restricted;
    private Boolean hr_access;

    public String getName() {
        return name;
    }
    public List<Integer> getUsers() {
        return users;
    }
    public List<Integer> getLocations() {
        return locations;
    }
    public List<Image> getImages() {
        return images;
    }
    public Integer getTokens() {
        return tokens;
    }
    public Boolean getRestricted() {
        return restricted;
    }
    public Boolean getHr_access() {
        return hr_access;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: ios/MyJobPitch/Utils/API/models/Business.swift
//
//  Business.swift
//  MyJobPitch
//
//  Created by dev on 12/20/16.
//  Copyright © 2016 myjobpitch. All rights reserved.
//

import Foundation

class Business: MJPObjectWithDates {
    
    static let mappingArray = [ "id", "users", "locations", "name", "tokens", "restricted", "hr_access" ]

    var name: String!
    var users: NSArray!
    var locations: NSArray!
    var images: NSArray!
    var tokens: NSNumber!
    
    var restricted = false
    
    var hr_access = false
    
    func getImage() -> Image? {
        if images != nil && images.count > 0 {
            return images.firstObject as? Image
        }
        return nil
    }
    
}
